CMakeLists.txt

```diff
@@ -52,27 +52,135 @@ target_link_libraries(logging_client PRIVATE
   http_parser.host
 )
 
-add_enclave_lib(loggingenc
-  SRCS
-    src/apps/logging/logging.cpp
-    src/apps/logging/stub_for_code_signing.cpp
-)
-sign_app_library(loggingenc
-  ${CMAKE_CURRENT_SOURCE_DIR}/src/apps/logging/oe_sign.conf
-  ${CMAKE_CURRENT_SOURCE_DIR}/src/apps/sample_key.pem
-)
+# Build common library for CCF enclaves
+add_custom_target(ccfcommon ALL)
+
+if("sgx" IN_LIST TARGET)
```

Comment:
We mostly "if (" rather than "if(" elsewhere in cmake. That reminds me to look for a formatter again.

Reply:
This looks plausible https://github.com/cheshirekow/cmake_format

Reply:
I've replaced `if(` with `if (`, but left `endif()` and `else()`, for consistency with `function(` and `endfunction(`. We should do a separate PR with a full CMake format pass if we adopt one.